src/Service/Lambda/README.md

```diff
@@ -10,3 +10,29 @@ An API client for Lambda.
 ```cli
 composer require async-aws/lambda
 ```
+
+## Usage
+
+````php
+$client = new LambdaClient([
+    Configuration::OPTION_REGION => 'us-east-1',
```

Comment:
I'm not sure we should expose consts `Configuration::OPTION_REGION` to userland. it's less readable IMHO

Reply:
Indeed it is less readable